pum/core/upgrader.py

```diff
@@ -477,7 +478,7 @@ def current_db_version(self):
 
         self.cursor.execute(query)
 
-        return self.cursor.fetchone()[0]
+        return pkg_resources.parse_version(self.cursor.fetchone()[0])
```

Comment:
pum is actually part of osgeo4w.
Do you know if pkg_resources is also bundled?

Reply:
pkg_resources comes with setuptools, so I guess it is.